### ISSUE TITLE
scrolling disabled on a pdf tab

### ISSUE THREAD

**Issue by sedghi** [philc/vimium#2498]
is scrolling (with j & f) disabled on a tab in which a pdf is opened? for example: 
http://www.pdf995.com/samples/pdf.pdf

**Comment by smblott-github** [comment]
Yup.  Because the PDF viewer is an embed, there's no API to to scroll or search the content.

**Comment by alirzsedghi** [comment]
thanks for the prompt response, I just found vimium, it is really great.
Just two more question: 
1) when I open a new tab with t , how can exit the insert mode in the url bar in order to use other shortcuts, grabbing next frame also does not work when I am in url bar
2) is there any way to use navigation in the text box?

**Comment by smblott-github** [comment]
1. See [here](https://github.com/philc/vimium/wiki/FAQ#how-can-i-get-vimium-to-work-on-chromes-new-tab-page).
2. I don't understand the question.  Which text box?

**Comment by alirzsedghi** [comment]
thanks , 
when pressing the *gi* , it goes to the first text input. My question is about whether there is any thing like normal mode for vim (moving around with w,e,b, .. ) inside text input

**Comment by smblott-github** [comment]
> My question is about whether there is any thing like normal mode for vim (moving around with w,e,b, .. ) inside text input

No, Vimium doesn't do that.  You might consider [wawavi](https://chrome.google.com/webstore/detail/wasavi/dgogifpkoilgiofhhhodbodcfgomelhe).

**Comment by alirzsedghi** [comment]
nima sedgh
